template/component-top-nav.html.jinja

```diff
@@ -1,9 +1,9 @@
-<div class="top-banner fixed-top">
+<!-- <div class="top-banner fixed-top">
```

Comment:
I suggest we put these in a Jinja comment so the commented-out version doesn't get built to the final site

```suggestion
<!--{# <div class="top-banner fixed-top">
```